packages/apps/tools/src/components/Common/Layout/partials/Header/Header.tsx

```diff
@@ -34,6 +35,7 @@ const Header: FC<IHeaderProps> = () => {
     e: React.MouseEvent<HTMLAnchorElement>,
   ): Promise<void> => {
     e.preventDefault();
+    console.log('e.currentTarget.href', e.currentTarget);
```

Comment:
Should we maybe add an eslint rule about `console` statements?

Reply:
We'll probably need the logs for places like these 
![Screenshot 2023-11-28 at 10 31 36](https://github.com/kadena-community/kadena.js/assets/132443209/0bcab9c4-38fa-435d-9ae0-eadae54ef294)

However, we can add some proper logger with all the levels we need

Reply:
Yes, but then in those specific cases, we could just disable the eslint rule for those lines, right?